srcpkgs/ruby-clocale/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'ruby-clocale'
+pkgname=ruby-clocale
+version=0.0.4
+revision=1
+noarch=yes
+build_style=gem
+makedepends="ruby-devel"
```

Comment:
This seems most certainly wrong. If you need ruby-devel then you are most likely compiling C extensions which are not noarch, and the build_style you're looking for is gemspec since we need to do a lot of hackery to get cross-compilation going.

Reply:
You are right. I will close this one and rework. I had this sitting around for a while and should've looked at it again before PR.